.changeset/@graphql-mesh_utils-9022-dependencies.md

```
@@ -0,0 +1,5 @@
+---
+"@graphql-mesh/utils": patch
+---
+dependencies updates:
+  - Updated dependency [`@graphql-tools/delegate@^12.0.0` ↗︎](https://www.npmjs.com/package/@graphql-tools/delegate/v/12.0.0) (from `^11.0.0`, in `dependencies`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)